### PR TITLE
DOC-1829-AddingCompression Level And Process Number to 8

### PR DESCRIPTION
**Pull request by JosephNewmanTG**
Addition/Update for 3.9.3
Attached to this task: https://graphsql.atlassian.net/browse/DOC-1829 

Changes in this pull request
- Added to table System.Backup.CompressionLevel, description, and default value
- Added to table System.Backup.DecompressProcessNumber, description, and default value
- Change default value of System.Backup.CompressProcessNumber from 0 to 10

Link to current tables in the documentation: 

1. https://docs.tigergraph.com/tigergraph-server/current/backup-and-restore/configurations#_configuration_parameters 
3. https://docs.tigergraph.com/tigergraph-server/current/reference/configuration-parameters#_system

### CODE REVIEW

File: modules/backup-and-restore/pages/configurations.adoc [hunk 1]
```diff
@@ -48,7 +48,14 @@ Typically used when operating in a private network or with a non-AWS S3-compatib
...
 It's recommended to keep the default value `0`, which means the number of processes used to compress is equal to the number of CPU cores on each node.
-| `0`
+| `8`
+
+|System.Backup.DecompressProcessNumber | The number of concurrent processes for decompression during the restore.
+| `8`
+
+|System.Backup.CompressionLevel |The backup compression level strikes a balance between size and speed. The better compression, the longer it takes.
+("BestSpeed", "DefaultCompression", "BestCompression")
+| `DefaultCompression`
```
Comment (victorleeTG) [inline modules/backup-and-restore/pages/configurations.adoc]:
```suggestion
|System.Backup.CompressionLevel
|One of three present levels of compression.  Higher compression takes longer. 
("BestSpeed", "DefaultCompression", "BestCompression")
| "DefaultCompression"
```

File: modules/backup-and-restore/pages/configurations.adoc [hunk 2]
```diff
@@ -48,7 +48,14 @@ Typically used when operating in a private network or with a non-AWS S3-compatib
 |System.Backup.CompressProcessNumber | Number of concurrent processes for compression during backup.
 
 It's recommended to keep the default value `0`, which means the number of processes used to compress is equal to the number of CPU cores on each node.
-| `0`
+| `8`
```
Comment (victorleeTG) [inline modules/backup-and-restore/pages/configurations.adoc]: Confirming: The default value is changing?  Note that that change would invalidate the preceding "It's recommended..."

File: modules/backup-and-restore/pages/configurations.adoc (hunk 1 above)
Comment (victorleeTG) [reply modules/backup-and-restore/pages/configurations.adoc]: The https://docs.tigergraph.com/tigergraph-server/current/reference/configuration-parameters is supposed to list ALL the configuration parameters.  This page (https://docs.tigergraph.com/tigergraph-server/current/backup-and-restore/configurations) may repeat a subset of those confirmation parameters, for convenience.

So, any change you make should me made in both places.  Please double check them for consistency.

**Review by victorleeTG — CHANGES_REQUESTED** [review]

File: modules/backup-and-restore/pages/configurations.adoc (hunk 1 above)
Comment (JosephNewmanTG) [reply modules/backup-and-restore/pages/configurations.adoc]: Thank you @victorleeTG I made these changes with my last commit.

File: modules/backup-and-restore/pages/configurations.adoc (hunk 2 above)
Comment (JosephNewmanTG) [reply modules/backup-and-restore/pages/configurations.adoc]: Yes as per the task. Will add the engineer/ticket reporter to confirm.

Comment (JosephNewmanTG) [reply modules/backup-and-restore/pages/configurations.adoc]: @runlilong can you please take a look at this PR and please confirm everything looks good.

Comment (runlilong) [reply modules/backup-and-restore/pages/configurations.adoc]: the default value has changed to 10. but still, value 0 means the number of processes used to compress equals the node CPU's cores.

File: modules/reference/pages/configuration-parameters.adoc [hunk 3]
```diff
@@ -1155,8 +1155,15 @@ empty. You can use @/cert/file/path to pass the certificate from a file.
 |`Va2V7mdpTY5ErZRmTBBRqYtkgR7CiGbF`
 
 |System.Backup.CompressProcessNumber |The number of concurrent process
-for compression during backup. Value 0 means the number of processes
-used to compress equals the node CPU’s cores. |`0`
+for compression during backup. Value 8 means the number of processes
```
Comment (runlilong) [inline modules/reference/pages/configuration-parameters.adoc]: same problem, value 0 is still supported, only it is not the default value anymore.

File: modules/backup-and-restore/pages/configurations.adoc (hunk 2 above)
Comment (JosephNewmanTG) [reply modules/backup-and-restore/pages/configurations.adoc]: Thank you both @runlilong and @victorleeTG I have committed this change. Default value is 10.

File: modules/reference/pages/configuration-parameters.adoc (hunk 3 above)
Comment (JosephNewmanTG) [reply modules/reference/pages/configuration-parameters.adoc]: Thank you @runlilong I have changed this with my latest commit. Default is now 10.

**Comment by JosephNewmanTG** [comment]
Resolved conversations, and made changes to PR. Merging to 3.9.3 staging branch.

**Pull request merged**